index.html

```diff
@@ -79,5 +80,14 @@ <h2 class="tech-title">Django</h2>
       </div>
     </div>
   </div>
+
+  <div class="footer">
+    <div class="footer-text-center">
+      <h1 class="footer-text-top">WE ARE GAME CHANGERS</h1>
```

Comment:
This is not that relevant to be a headline, you should use a span since is a short piece of text